### DISCUSSION TITLE
Running npm ci like suggested in the readme.md shows that inflight@1.0.6 leaks of memory and should not be used, but package.json doesn't contain it

### DISCUSSION THREAD

**Discussion by Umeaboy** [tutao/tutanota#7675] [General]
- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Describe the bug**
When running npm ci in the main directory I see this:

npm warn skipping integrity check for git dependency ssh://git@github.com/tutao/better-sqlite3-sqlcipher.git
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @humanwhocodes/config-array@0.11.14: Use @eslint/config-array instead
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated electron-packager@17.1.2: Please use @electron/packager moving forward. There is no API change, just a package name change

I agree that they are only warnings, but if the app leaks memory it will affect the usage of the Destkop user.

**To Reproduce**
1. Clone https://github.com/tutao/tutanota.git
2. cd to the cloned tutanota directory.
3. Run npm ci
4. These warnings appears as soon as npm has started working.

**Expected behavior**
It should run without warnings.

**Desktop (please complete the following information):**
 - OS: Mageia 9
 - Desktop Environment: Cinnamon
 - Version: 5.6.1

**Comment by charlag** [comment]
It is a transitive dependency.

**Reply by Umeaboy** [reply]
> It is a transitive dependency.

Sorry for sounding like a complete newbie, but where is that called from? Seems stupid to have a warning about a line somewhere if it isn't even used.

No offence meant.

**Reply by charlag** [reply]
It's not called directly from anywhere. If you look for `inflight` in `package-lock.json` (or in `npm list --all`) you can find that it is ultimately a dependency of `electron-builder`. The warning is absolutely useless to us but that's how npm works.
